docs/spark-qualification-tool.md

```diff
@@ -215,28 +229,39 @@ Note: The “regular expression” used by -a option is based on
 [java.util.regex.Pattern](https://docs.oracle.com/javase/8/docs/api/java/util/regex/Pattern.html).
 
 ### The Qualification tool output
-After the above command is executed, the summary report goes to STDOUT and by default it outputs 2 files 
-under `./rapids_4_spark_qualification_output/` that contain the processed applications.
+After the above command is executed, the summary report goes to STDOUT and by default it outputs
+log/CSV files under `./rapids_4_spark_qualification_output/` that contain the processed applications.
 The output will go into your default filesystem and it supports both local filesystem and HDFS. 
 Note that if you are on an HDFS cluster the default filesystem is likely HDFS for both the input and output.
-If you want to point to the local filesystem be sure to include `file:` in the path
+If you want to point to the local filesystem be sure to include `file:` in the path.
+
+The Qualification tool generates two main outputs to help quantify the expected acceleration of migrating a Spark
```

Comment:
it might be nice to separate out the output location and files vs the fields within the output.  Perhaps put the information about files first and then go over the fields within it, or maybe we reference the understanding the qualification tool output section below and maybe this section becomes "output location" or something like that